phpcs.xml

```diff
@@ -20,4 +20,32 @@
 			</property>
 		</properties>
 	</rule>
+
+	<!-- Run phpcs on current directory -->
+	<file>.</file>
+
+	<!-- Show progress, show the error codes for each message (source). -->
+	<arg value="ps"/>
+
+	<!-- Strip the filepaths in reports down to the relevant bit. -->
+	<arg name="basepath" value="./"/>
+
```

Comment:
```suggestion
```

In my experience this can cause problems for IDE integrations. The path is reported relative to the project but the IDE extensions expect the path to be complete. It may be extension dependent.